Sources/SwiftInspectorVisitors/Tests/ProtocolVisitorSpec.swift

```diff
@@ -191,6 +191,23 @@ final class ProtocolVisitorSpec: QuickSpec {
               expect(protocolInfo?.innerTypealiases.first?.initializer?.asSource) == "Any"
             }
           }
+
+          context("with properties") {
+            it("finds the typealias") {
```

Comment:
copy/paste? I don't understand this `it` since nothing in this test has to do with typealiases

Reply:
https://github.com/fdiaz/SwiftInspector/pull/100/commits/4f14f652851b23219a06e580d10e013a49c1a494